ACE/ace/Message_Queue_T.h

```diff
@@ -669,12 +669,13 @@ class ACE_Message_Queue_Iterator
   /// Declare the dynamic allocation hooks.
   ACE_ALLOC_HOOK_DECLARE;
 
+protected:
+  /// Keeps track of how far we've advanced...
+  ACE_Message_Block *curr_;
```

Comment:
Since this changes the declaration order of the data members, the order that those members appear in the constructor's member initializer list needs to change as well.

Reply:
done